scripts/index.js

```diff
@@ -42,4 +42,18 @@ function coolCities(citiesArray) {
 })
 return coldCities
 }
-coolCities(cities)
+coolCities(cities) 
+
+function cityNames(citiesArray) {
+    let cityNameArray = []
+    let allCities = Object.values(citiesArray);
```

Comment:
Hmmm...I'm not sure you need `Object.values`, since `citiesArray` does not have key/value pairs? You should be able to `.forEach` through `citiesArray` without it.